src/engine/map_renderer.cpp

```diff
@@ -287,17 +287,35 @@ void MapRenderer::switchBackdrops()
 
 void MapRenderer::renderBackground(
   const base::Vec2& sectionStart,
-  const base::Size& sectionSize) const
+  const base::Size& sectionSize,
+  const float backColorMod) const
 {
+  if (backColorMod < 1.0f)
```

Comment:
You can simplify this a little bit by doing:

```cpp
const auto saved = renderer::saveState(mpRenderer);
```

Before setting the color mod. The previous value is then automatically restored when the function returns.
So you don't need the second `if` + `setColorMod` then.

Reply:
Unfortunately, this method didn't seem to work for me.

Reply:
I'll have a look

Reply:
> Thanks for that. I figured there was something I was missing. Visual Studio recognized the .clang-format file, but obviously it wasn't quite doing the trick (at least the way it's currently configured for me).

Visual Studio's clang-format integration doesn't work for me either. They most likely have a different version of the formatter bundled. Unfortunately, with clang-format, the configuration file is not enough - the version also needs to match